libqtile/widget/groupbox.py

```diff
@@ -285,10 +285,14 @@ def draw(self):
             else:
                 text_color = self.inactive
 
-            if g.screen:
+            if g.screen == self.bar.screen:
```

Comment:
It might be good to add an `elif g.screen` to configure the color of the group on the other screen, see the `else` on L292 below. Also that logic in the `else` block can then be condensed